app/upgrades/v25/constants.go

```diff
@@ -1,19 +1,37 @@
 package v25
 
 import (
+	"github.com/osmosis-labs/osmosis/osmomath"
 	"github.com/osmosis-labs/osmosis/v24/app/upgrades"
 
 	store "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	auctiontypes "github.com/skip-mev/block-sdk/x/auction/types"
 )
 
 // UpgradeName defines the on-chain upgrade name for the Osmosis v25 upgrade.
-const UpgradeName = "v25"
+const (
+	UpgradeName = "v25"
+
+	// Noble USDC is used as the auction denom
+	AuctionUSDCDenom = "ibc/498A0751C798A0D9A389AA3691123DADA57DAA4FE165D5C75894505B876BA6E4"
```

Comment:
Please double check this is correct :pray:

Reply:
confirmed